flocker/node/test/test_deploy.py

```diff
@@ -619,7 +619,9 @@ def test_does_not_exist(self):
     name=APP_NAME2,
     image=DockerImage.from_string(UNIT_FOR_APP2.container_image)
 )
-EMPTY_NODESTATE = NodeState(hostname=u"example.com")
+EMPTY_NODESTATE = NodeState(hostname=u"example.com",
+                            manifestations={}, applications=[],
+                            used_ports=[], devices={}, paths={})
```

Comment:
Also passing in uuid will break a bunch of tests that assuming just having `hostname="example.com"` is sufficient.